specification.md

```diff
@@ -3841,9 +3909,9 @@ This allows you to test if your workflow behaves properly for cases when there a
 | id | Unique state id | string | no |
 | name | State name | string | yes |
 | type | State type | string | yes |
-| inputCollection | Workflow expression selecting an array element of the states data | string | yes |
-| outputCollection | Workflow expression specifying an array element of the states data to add the results of each iteration | string | no |
-| iterationParam | Name of the iteration parameter that can be referenced in actions/workflow. For each parallel iteration, this param should contain an unique element of the inputCollection array | string | yes |
+| inputCollection | Array or a workflow expression creating an array | array or expression | yes |
+| outputCollection | Workflow expression specifying a path where an array is created that collects the results of each iteration | path expression | no |
```

Comment:
whats a "path" in this sence. 
also outputcollection does not have to be created..it can exist already

Reply:
that's the complication we have from not passing the output to the expression.

Reply:
Ok, as with jsonpointer, removed the path expression thingy to not have them in the spec